batavia/types/Complex.js

```diff
@@ -3,15 +3,19 @@ var exceptions = require('../core').exceptions
 var version = require('../core').version
 var type_name = require('../core').type_name
 var create_pyclass = require('../core').create_pyclass
+var Int = require('./Int.js')
 var NotImplemented = require('../core').NotImplemented
 
 // Helper function defined in Float.js
 var scientific_notation_exponent_fix = require('./Float').scientific_notation_exponent_fix
-
 /*************************************************************************
  * A Python complex type
  *************************************************************************/
 
+var MAX_INT = new Int('9223372036854775807')
+var MAX_FLOAT = new Int('179769313486231580793728971405303415079934132710037826936173778980444968292764750946649017977587207096330286416692887910946555547851940402630657488671505820681908902000708383676273854845817711531764475730270069855571366959622842914819860834936475292719074168444365510704342711559699508093042880177904174497791')
+var MIN_FLOAT = new Int('-179769313486231580793728971405303415079934132710037826936173778980444968292764750946649017977587207096330286416692887910946555547851940402630657488671505820681908902000708383676273854845817711531764475730270069855571366959622842914819860834936475292719074168444365510704342711559699508093042880177904174497791')
```

Comment:
Aren't these three constants already defined in the definitions of Int and Float respectively?

Reply:
That they are. Good call.